spec/lib/job_notifier/notifier_spec.rb

```diff
@@ -68,7 +70,7 @@ def perform_with_feedback(param1, param2)
 
     context "with error feedback" do
       before do
-        class ImageUploadJob < ActiveJob::Base
+        class ImageUploadJob < ApplicationJob
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/ConstantDefinitionInBlock: Do not define constants this way within a block.